docs/free-trial.rst

```diff
@@ -0,0 +1,103 @@
+.. _free-trial:
+
+==========
+Free trial
+==========
+
+If you want to try out CrateDB Cloud's potential, there are multiple ways to do
+this. One is to subscribe to one of the CrateDB Cloud offers via our SaaS
+marketplace offers and to choose the `Development subscription plan`_. For a
+full guide on how to subscribe via the marketplace offers, see our
+:ref:`tutorials on deploying a cluster from scratch <sign-up>` above.
+
+The other is to avail yourself of the possibility of our free trial: a fully
+functioning CrateDB Cloud cluster running for 14 days without cost, so you can
+experience the benefits of CrateDB Cloud without any immediate commitment.
+
+In this guide, we describe step by step how to activate the CrateDB Cloud free
+trial offer. The CrateDB Cloud structure is based on organizations, and within
+these organizations run projects that contain the deployed clusters. To deploy
+your free trial cluster, you therefore need an organization and a project to
+deploy it in as well. Fortunately, these are created quite easily as part of
+the deployment process, as will be shown in the following steps.
+
+
+.. _free-trial-signup:
+
+Sign up
+=======
+
+In order to access CrateDB Cloud, you must first sign up with an account.
+Simply follow the steps in the :ref:`tutorial for signing up <sign-up>`. When
+you have successfully signed up, you will be forwarded to the CrateDB Cloud
+Console subscription overview. It is from here that you can make use of the
+free trial option. The first step is to set up an organization.
+
+
+.. _free-trial-org:
+
+Create an organization
+======================
+
+Before the free trial can be deployed, it is necessary to create an
+organization within the CrateDB Cloud Console. The trial cluster is then
+deployed within this organization. When you arrive for the first time at the
+Console subscription overview, you should be prompted to create an
+organization.
+
+.. image:: _assets/img/free-trial-organization.png
```

Comment:
I think we should make use of alternative texts to the images:

```suggestion
.. image:: _assets/img/free-trial-organization.png
   :alt: Create an organization
```

Reply:
Is the image automatically sized to 100% width?

Reply:
It is if it is oversize, I don't think it is otherwise. I agree re: alt text - we've not done that anywhere so far, but it could be a good practice